### ISSUE TITLE
Handle package-internal symbols

### ISSUE THREAD

**Issue by joostkremers** [Malabarba/Nameless#2]
Pretty cool package.

Would it be possible to add another prefix for package-internal symbols? Those that start with the package prefix followed by two dashes? Right now I have things like `:next-field` together with `:-redisplay` in my code. IHMO it'd be nice if the latter would look like `::redisplay`.

**Comment by Malabarba** [comment]
Hmm. I think it's possible.
I'll try to figure out the best way to do this.

**Comment by Malabarba** [comment]
Done. Should be up tomorrow. Do `(setq nameless-private-prefix t)` to enable this.

**Comment by joostkremers** [comment]
I set `nameless-private-prefix` to `t` and then was surprised by the fact that it didn’t work... Turns out it was because I had `nameless-affect-indentation-and-filling` set to `nil`, which causes the variable `compose` in `nameless--compose-as` to be set to `nil`, so that the entire following `when` clause isn’t executed.

BTW, small nitpick: in the definition of `nameless-affect-indentation-and-filling`, there seems to be an erroneous quote before `outside-strings` in the `:type` definition. In the Customization buffer, the value is marked as a mismatch. The value of the `:type` specification is quoted already.
